apps/src/code-studio/initApp/project.js

```diff
@@ -193,6 +202,32 @@ var projects = module.exports = {
    * @return {boolean} true if we should show our abuse box instead of showing
    *   the project.
    */
+  hideBecausePrivacyViolationOrProfane: function () {
+    if (!this.hasPrivacyProfanityViolation() || appOptions.scriptId) {
```

Comment:
I might swap the order of these, as I think the first is the more important one.